uber/templates/regform.html

```diff
@@ -285,10 +285,36 @@
 <div class="form-group">
     <label for="email" class="col-sm-2 control-label">Email Address</label>
     <div class="col-sm-6">
-        <input type="email" name="email" id="email" value="{{ attendee.email }}" class="form-control" placeholder="Email Address">
+        <input type="email" name="email" id="email" value="{{ attendee.email }}" class="form-control" placeholder="Email Address" id="mailValue1">
     </div>
 </div>
 
+<div class="form-group">
+    <label for="email" class="col-sm-2 control-label">Email Address Confirmation</label>
+    <div class="col-sm-6">
+        <input type="email" name="email2" id="email2" value=" emailValue2" class="form-control" placeholder="Email Address 2" id="mailValue2">
+    </div>
+</div>
+
+<div hidden class="email-error">
+        <br>
+    <p style="text-indent: 23em; color: red"><b>THE EMAIL Addresses entered do not match, please enter them again.</b></p>
+</div> <br>
+<div hidden class="email-identical">
+        <br>
+    <p style="text-indent: 23em; color: green"><b>THE EMAIL Addresses entered match.</b></p>
+</div> <br>
+
+<script>
+    var emailValue2;
+        if(document.getElementById("mailValue2").value == document.getElementById("mailValue1").value){
+            document.getElementsByClassName("email-identical").removeAttribute("hidden");
+        }
+        else{
+            document.getElementsByClassName("email-error").removeAttribute("hidden");
+        }
+</script>
```

Comment:
couple of suggestions here.  

1) this code is the right logic, though I believe this will execute just one time.  you would need to hook this into an event handler that is called when either of the email fields has their text changed.  check out some of the "onChange" sections in this file for examples, or check out a jQuery tutorial for some more examples there.

2) there's easier more cross-platform ways to change these elements using jQuery.  check out some of the other javascript in regform.html, such as the code around donationChanged();.   you can replace the document.getElement* with the jquery code. (look for the function calls with the dollar signs)

getting close!